_news/174.md

```diff
@@ -0,0 +1,136 @@
+  `[A+B || A <- [1,2,3] && B <- [4,5,6]]` will produce `[5,7,9]`.
+  
+- Generators in comprehensions can now be strict, meaning that if
+  the generator pattern does not match, an exception will be raised
+  instead of silently ignore the value that didn't match.
+
+- It is now possible to use any base for floating point numbers as per
+  [EEP 75: Based Floating Point Literals](https://www.erlang.org/eeps/eep-0075).
+
+
+## Compiler and JIT improvements
+
+- For certain types of errors, the compiler can now suggest
+  corrections.  For example, when attempting to use variable `A` that
+  is not defined but `A0` is, the compiler could emit the following
+  message: `variable 'A' is unbound, did you mean 'A0'?`
+
+- The size of an atom in the Erlang source code was limited to 255
+  bytes in previous releases, meaning that an atom containing only
+  emojis could contain only 63 emojis. While atoms are still only
+  allowed to contain 255 characters, the number of bytes is no longer
+  limited.
+
+- The `warn_deprecated_catch` option enables warnings for use of
+  old-style catch expressions on the form `catch Expr` instead of the
+  modern `try` ... `catch` ... `end`.
+  
+- Provided that the map argument for a `maps:put/3` call is known to
+  the compiler to be a map, the compiler will replace such calls with
+  the corresponding update using the map syntax.
+  
+- Some BIFs with side-effects (such as `binary_to_atom/1`) are
+  optimized in `try` ... `catch` in the same way as guard BIFs in
+  order to gain performance.
+
+## ERTS
+
+- The `trace:system/3` function has been added. It has a similar
+  interface as `erlang:system_monitor/2` but it also supports trace
+  sessions.
+
+- `os:set_signal/2` now supports setting handlers for the `SIGWINCH`,
+  `SIGCONT`, and `SIGINFO` signals.
+  
+- The two new BIFs `erlang:processes_iterator/0` and
+  `erlang:process_next/1` make it possible to iterate over the
+  process table in a way that scales better than `erlang:processes/0`.
+
+## Shell and terminal
+
+- The erl -noshell mode has been updated to have two sub modes called
+  `raw` and `cooked`, where `cooked` is the old default behaviour and
+  `raw` can be used to bypass the line-editing support of the native
+  terminal. Using `raw` mode it is possible to read keystrokes as they
+  occur without the user having to press Enter. Also, the `raw` mode
+  does not echo the typed characters to stdout.
+  
+- The shell now prints a help message explaining how to interrupt a
+  running command when stuck executing a command for longer than 5
+  seconds.
+
+## STDLIB
+
+- The `join(Binaries, Separator)` function that joins a list of
+  binaries has been added to the
+  [`binary`](https://www.erlang.org/doc/apps/stdlib/re.html) module.
+
+- By default, sets created by module
+  [`sets`](https://www.erlang.org/doc/apps/stdlib/sets.html)
+  will now be represented as maps.
+  
+- Module [`re`](https://www.erlang.org/doc/apps/stdlib/re.html) has
```

Comment:
Should this link (and others) point to the new documentation for rc1?

Reply:
Yes, ideally, but I am not sure where this documentation will be. Maybe it's easier to adjust the links when the release has been built? Any thoughts, @proxyles or @garazdawi?

Reply:
As the rc1 docs will have the version number of the app in them, it is easier to just update when the docs are published.